docs/pages/toolpad/core/api/account.json

```diff
@@ -1,13 +1,21 @@
 {
   "props": {
-    "signInLabel": { "type": { "name": "string" }, "default": "'Sign In'" },
-    "signOutLabel": { "type": { "name": "string" }, "default": "'Sign Out'" },
+    "localeText": {
+      "type": { "name": "shape", "description": "{ signInLabel: string, signOutLabel: string }" },
+      "default": "DEFAULT_LABELS"
```

Comment:
Does MUI X also list constants as defaults like this, even if they're not accessible for users?
Just curious, as not sure if there's a better way, but if it's following an existing pattern I guess it's fine!

Reply:
Yep: https://github.com/mui/mui-x/blob/master/packages/x-data-grid/src/constants/localeTextConstants.ts